cmd/relayer/main.go

```diff
@@ -0,0 +1,412 @@
+package main
```

Comment:
Can you add package comments.  See swapd's main as as example. Just a brief description of what the relayer is, and a mention that this is only the entrypoint for the relayer (defining the package's scope).

Reply:
updated!